Headers/DebugServer2/Types.h

```diff
@@ -106,6 +106,11 @@ struct StopInfo {
     kReasonBreakpoint,
     kReasonSignalStop, // TODO better name
     kReasonTrap,
+#if defined(OS_WIN32)
+    kReasonMemoryError,
+    kReasonMathError,
+    kReasonInstructionError,
```

Comment:
Could we make kReasonException which mirrors kReasonTrap (with an exception code member variable mirroring `signal`)?